ST_params.c

```diff
@@ -1090,8 +1048,8 @@ static void _species_init( void) {
    Bool readspp = TRUE, sppok = TRUE;
 
    /* temp vars to hold the group info*/
-   char name[80], /* plenty of space to read possibly long name*/
-        name2[80]; /* another one*/
+   char *name, *name2;
```

Comment:
@chaukap They were there before I started working with the function, so I just kept them. I can check the function to see if we can get rid of the second name variable.

Reply:
@chaukap Will do the same for other related changes you have suggested. For the most part, the code was already like that before I started working with it.

Reply:
@fdpierson I totally understand! It was definitely a redundancy added by a previous programmer, but I think it would be nice to clean it up here before merging.

Reply:
@chaukap 4546596 should resolve this issue and other similar to it.